### PR TITLE
Postgres/Cassandra

### PR DESCRIPTION
**Pull request by Nguyephi**

### CODE REVIEW

**Review by azdoolittle — APPROVED** [review]
lgtm

File: generateData/csdbSchema.cql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+CREATE KEYSPACE IF NOT EXISTS airbnb
+WITH REPLICATION = {
```
Comment (priscilagonzalez) [inline generateData/csdbSchema.cql:2]: Does this line have to do with distributed systems?

**Review by priscilagonzalez — COMMENTED** [review]
Nice! Lgtm.

**Review by kraig-droid — COMMENTED** [review]
lgtm

File: generateData/csdbSchema.cql (hunk 1 above)
Comment (Nguyephi) [reply generateData/csdbSchema.cql:2]: I'm not sure what you mean by distributed systems

Comment (kraig-droid) [reply generateData/csdbSchema.cql:2]: Yes, Cassandra has a "replication factor" that says how many "nodes" should have a copy of the data.

**Pull request merged**